lib/chatBot.js

```diff
@@ -712,26 +734,52 @@ ChatBot.prototype._onLogOnResponse = function(res) {
 					sessionID: sessionid,
 					webCookie: cookies
 				}, function(e, api) {
-					if(e) { that.winston.error('ChatBot ' + that.name + ' error getting API key: ' + e); }
+					if(e) { that.winston.error(that.name + '/chatBot.js: error getting API key: ' + e); }
 					else { that.apikey = api; }
-			  });
-			  that.winston.info('ChatBot ' + that.name + ': logged into Steam web');
+			});
+			that.winston.info(that.name + '/chatBot.js: logged into Steam web');
 			}
 		});
+	} else if(res.eresult === 63) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/969820410/issues/source?bid=1980036&fileBranchId=2848073#l744)